heron/storm/src/java/org/apache/storm/generated/ShellComponent.java

```diff
@@ -0,0 +1,46 @@
+// Copyright 2016 Twitter. All rights reserved.
```

Comment:
If this file is written by you, this copyright is fine or if the file is taken from Apache Storm, please use their copyright.